### ISSUE TITLE
Add an option to choose what to show in the video call

### ISSUE THREAD

**Issue by wuniversales** [qTox/qTox#5964]
It would be great to be able to share the desktop in the video call.

**Comment by anthonybilinski** [comment]
What OS are you on? On Linux X11, under settings -> audio/video, "Desktop" should be an option, with the ability to choose which monitor and select a region. On Wayland screen capture is not supported.

**Comment by wuniversales** [comment]
I use linux (Manjaro)

True, there is the option, I just realized, sorry.

When you start a video call ...
Can't you add an option to choose what to show in the video call?
If you show the desktop, or the camera.
Something like a drop-down menu in the video call.

I think it would be better that way, right? A "Quick access" of the camera configuration.

**Comment by anthonybilinski** [comment]
Yeah that sounds like a nicer UI experience to switch devices in call. Or at least a link that quickly takes you to the settings section.
